### ISSUE TITLE
Can we add the note like 4.8/5

### ISSUE THREAD

**Issue by Dracula06** [Der-Henning/tgtg#88]
Can we add the note on the telegram notification ?

2022-05-10 14:16:03 INFO     Sending notifications for Casino Supermarché - Cagnes Sur Mer - **4.8/8** - 1 bags available
instead of
2022-05-10 14:16:03 INFO     Sending notifications for Casino Supermarché - Cagnes Sur Mer - 1 bags available

**Comment by Dracula06** [comment]
Casino Supermarché - Cagnes Sur Mer
Available: 1
Price: 3.99 EUR
Note: 4.8/5
Pickup: Tomorrow, 11:00 - 14:00

instead of

Casino Supermarché - Cagnes Sur Mer
Available: 1
Price: 3.99 EUR
Pickup: Tomorrow, 11:00 - 14:00

**Comment by Der-Henning** [comment]
Hi @Dracula06.
Thank you for your suggestion. 
I propose an additional option for telegram notifier where you can define message body similar to webhook notifier.
I try to add this in the next iteration, but it will take some time. 
Feel free to create a fork and implement it yourself and maybe create a PR.

**Comment by Dracula06** [comment]
I'm sorry but i can't do anything, i m really noob in coding. Thanks if you do it, amazing job.

**Comment by Dracula06** [comment]
Sorry to bother you but can you add the possibility to add a link like the android feature.

Like that : https://share.toogoodtogo.com/item/685092/

the number is the item_id in the metrics variable i think

`tgtg_send_notifications_created{display_name="Les Délices de la Lanterne (Panier Patisserie)",**item_id**="**685092**"} 1.6526340908694625e+09`

So we can adjust in telegram message like that : 

```
Copenhagen Coffee Lab - Villeneuve-Loubet
Note: 4.2/5
Available: 2
Price: 3.99 EUR
Pickup: Today, 14:30 - 15:00
Link here: https://share.toogoodtogo.com/item/685092/
```

**Comment by Der-Henning** [comment]
Hi @Dracula06.
Yes this would be possible when I implement the described feature. I am confident that I can release a new version this week.
This feature was already requested by @Anonymousey in #85.

**Comment by Dracula06** [comment]
Thanks a lot man, you a beast!

**Comment by Der-Henning** [comment]
With [1e44cd8](https://github.com/Der-Henning/tgtg/commit/1e44cd882d47231e09f991cdd3ce7f7ab10bf9c8) I added the described functionality. It is already build into the latest docker image. 
Description in `docker-compose.yml`.

@Dracula06 are you using docker or the compiled desktop releases? If you are on docker I would appreciate if you can test and give feedback before I upload the next release.

**Comment by Dracula06** [comment]
I don t know how to build with the source but send me a mail with the compiled windows one and i can test it if you need.
My mail is Lecodebarre@gmail.com 
Erase my mail from comment when you already took it thanks :)

**Comment by Der-Henning** [comment]
I published the new Release v1.10.0 containing the requested features. Closing.

**Comment by Dracula06** [comment]
Hey BRO

I have some errors and i don t know how to fix that.
I have 2 telegram channels
The first one is ok
But the second one, each notification comes twice.

My config.ini without my tgtg credentials :

[MAIN]
Debug = false
SleepTime = 60
ScheduleCron =
ItemIDs =
Metrics = true
MetricsPort = 8000

[TGTG]
Username =
AccessToken =
RefreshToken =
UserId =

[SMTP]
enabled = false
Host = smtp.gmail.com
Port = 465
Username = ***@***.***
Password =
TLS = true
Sender = ***@***.***
Recipient = ***@***.***

[PUSHSAFER]
enabled = false
Key =
DeviceID =

[IFTTT]
enabled = false
Event =
Key =

[TELEGRAM]
enabled = true
token = 5236830281:AAGjV4obRydCTRyfex2KfvOY8pFESYviHrE
chat_id = -1001505696356
body = *${{display_name}}*\n*Disponible*: ${{items_available}}\n*Note sur 5*: ${{rating}}\n*Prix*: ${{price}} ${{currency}}\n*Pickup*: ${{pickupdate}}\n*Link*: https://share.toogoodtogo.com/item/${{item_id}}
chat_ids = 1115699127,-1001505696356

[WEBHOOK]
enabled = false
URL =
Method = POST
body =
type = text/plain
timeout = 60
